src/main/java/org/broadinstitute/hellbender/tools/walkers/coverage/CoverageOutputWriter.java

```diff
@@ -61,7 +62,8 @@
     private DEPTH_OF_COVERAGE_OUTPUT_FORMAT outputFormat;
     private boolean omitDepthOutput;
     private List<Integer> coverageThresholds;
-    final static DecimalFormat DOUBLE_FORMAT = new DecimalFormat("0.00");
+    private final DecimalFormat DOUBLE_FORMAT_2PLACES = new DecimalFormat("0.00");
```

Comment:
these should technically probably be lowercase, but eh

Reply:
I leave them uppercase to show my excitement for decimal formatting...